public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -219,6 +219,7 @@ export class PanelQueryRunner {
 
     const ctx: DataTransformContext = {
       interpolate: (v: string) => this.templateSrv.replace(v, data?.request?.scopedVars),
+      featureToggles: config.featureToggles,
```

Comment:
the only problem with this is that transformDataFrame is called from other places (plugins, scenes). 

But I checked pluginImports and could not find any plugin that imports transformDataFrame, which was surprising 

https://console.cloud.google.com/bigquery?sq=194555723165:e98021123ddc4719acc2d8b7b2e61074&pli=1&project=raintank-dev&authuser=1&ws=!1m8!1m7!8m2!1s194555723165!2se98021123ddc4719acc2d8b7b2e61074!14m3!1sraintank-dev!2sbquxjob_1554a026_18c1f9a6d6e!3sUS

thoughts @dprokop @ryantxu  ? 

I wonder if need more long term solution, can we call a function from here: 
https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/config.ts#L266

that sets a config object (or just feature toggles object) that lives in data?

Reply:
> I wonder if need more long term solution, can we call a function from here:
https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/config.ts#L266
>
> that sets a config object (or just feature toggles object) that lives in data?

I can give it a try. I'll close this PR for now.

Reply:
@torkelo would something like this work: https://github.com/grafana/grafana/pull/78904